tests/test_evaluations/test_evaluation_functions.py

```diff
@@ -117,6 +117,31 @@ def test_evaluation_list_per_fold(self):
             self.assertIsNotNone(evaluations[run_id].value)
             self.assertIsNone(evaluations[run_id].values)
 
+    def test_evaluation_list_sort(self):
+        openml.config.server = self.test_server
+        size = 10
+        task_id = 1769
+        # Get all evaluations of the task
+        unsorted_eval = openml.evaluations.list_evaluations(
+            "predictive_accuracy", offset=0, task=[task_id])
+        # Get top 10 evaluations of the same task
+        sorted_eval = openml.evaluations.list_evaluations(
+            "predictive_accuracy", size=size, offset=0, task=[task_id], sort_order="desc")
+
+        sorted_output = []
+        unsorted_output = []
+        for run_id in sorted_eval.keys():
+            sorted_output.append(sorted_eval[run_id].value)
+        for run_id in unsorted_eval.keys():
+            unsorted_output.append(unsorted_eval[run_id].value)
+
+        # Check if output from sort is sorted in the right order
+        self.assertTrue(sorted(sorted_output, reverse=True) == sorted_output)
```

Comment:
I don't think this works. I think the `sorted_output` in this case would contain `OpenMLEvaluation` objects which do not have a built-in comparison methods. You'll have to specify how you want to sort them, (e.g. specify a function through the `key` parameter).

Reply:
sorted_output contains a list of values of the score. So its a list that looks like [1,0.98,0.9....].
(I checked these when I did this on the production server.)

Reply:
My bad. I read it wrong :) I didn't notice you mapped to the `value` property of evaluations.

Reply:
Yep, my bad. I missed you mapped to the `OpenMLEvaluation.value` and not the `OpenMLEvaluation`.